js/dashboard-demo/dashboard-updates.js

```diff
@@ -548,4 +554,3 @@ setInterval(function () {
     }
 
 }, 5000);
-
```

Comment:
Whitespace removed

Reply:
What do you mean by whitespace removed I don't get it

Reply:
The new line in the end of the file has been removed.

Reply:
On my code page it's not removed, it's as the original page I just check, there is nothing

Reply:
Ok there is a empty line less you're right I put it back